cmd/neofs-cli/modules/acl/extended/create.go

```diff
@@ -41,7 +41,8 @@ Target is
   'user' for container owner, 
   'system' for Storage nodes in container and Inner Ring nodes,
   'others' for all other request senders, 
-  'pubkey:<key1>,<key2>,...' for exact request sender, where <key> is a hex-encoded 33-byte public key.
+  'pubkey:<key1>,<key2>,...' for exact request sender, where <key> is a hex-encoded 33-byte public key,
+  'account:<acc1>,<acc2>,...' for exact request sender, where <acc> is a base58 25-byte account.
```

Comment:
i think we need to note that it is our account somehow (mb reference to user ID or say that is it neo-go's string address)
@roman-khimov

Reply:
Stringified they're the same, we can in fact just say that it's an address.

Reply:
And it'd be preferable, `NSiVJYZej4XsxG5CUpdwn7VRQk8iiiDMPM` is widely known as _address_.

Reply:
Replaced account with address. Added example